controlplane/src/core/util.ts

```diff
@@ -187,6 +187,8 @@ export const getHighestPriorityRole = ({ userRoles }: { userRoles: string[] }) =
 };
 
 export const isValidOrganizationSlug = (slug: string): boolean => {
+  const reservedSlugs = ['login', 'signup', 'create', 'account'];
```

Comment:
Good catch. How can we prevent more in the future? I think we should introduce a different path prefix.